test/app-luatest/gh_7031_configure_eos_in_lua_console_test.lua

```diff
@@ -0,0 +1,88 @@
+local server = require('luatest.server')
+local socket = require('socket')
+local fio = require('fio')
+local t = require('luatest')
+local g = t.group()
+
+local empty_default_local_eos_res = [[tarantool> \set output lua
+true
+tarantool> 42
+42
+tarantool> ]]
```

Comment:
It is a bit complicate to read the test, when a request and a response are in different places. How about using the interactive_tarantool.lua helper? It seems it has everything needed for the test (except a prompt setter).

Please, take a look at https://github.com/Totktonada/tarantool/commit/f6ed51758e2dfdcdd5e047e5afa69035bb27d154.

Reply:
Done.